crates/swc_css_parser/src/lexer/mod.rs

```diff
@@ -119,10 +154,8 @@ where
             }
             // U+0023 NUMBER SIGN (#)
             Some(c) if c == '#' => {
```

Comment:
I found another issue.
It should be `Some('#')`, not `Some(c) if c == '#'`.

But this is not related to this PR, so let's skip it.

Reply:
Good catch I will fix it